SSO-Auth/Api/SSOController.cs

```diff
@@ -246,6 +246,15 @@ public async Task<ActionResult> OIDAuth([FromBody] AuthResponse response)
                     {
                         var authenticationResult = await Authenticate(kvp.Value.Username, kvp.Value.Admin, config.EnableAuthorization, config.EnableAllFolders, kvp.Value.Folders.ToArray(), response)
                             .ConfigureAwait(false);
+                            
+                        if(config.DefaultProvider != "")
```

Comment:
This should not be here, and instead should be moved into the `Authenticate` function.